javascript-if/main.js

```diff
@@ -0,0 +1,63 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  if (number < 5) {
+    return true;
+  } else { return false; }
+}
+
+function isEven(number) {
+  if (number % 2 === 0) {
+    return true;
+  } else { return false; }
+}
+
+function startsWithJ(string) {
+  if (string[0] === 'J') {
+    return true;
+  } else { return false; }
+}
+
+function isOldEnoughToDrink(person) {
+  if (person.age >= 21) {
+    return true;
+  } else { return false; }
+}
+
+function isOldEnoughToDrive(person) {
+  if (person.age >= 16) {
+    return true;
+  } else { return false; }
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  return false;
+}
+
+function categorizeAcidity(ph) {
+  if (ph < 0) {
+    return 'invalid pH level';
+  } else if (ph < 7) {
+    return 'acid';
+  } else if (ph === 7) {
+    return 'neutral';
+  } else if (ph <= 14) {
+    return 'base';
+  } else { return 'invalid pH'; }
+}
```

Comment:
Make sure pH less than zero and greater than 14 return the exact same message:

---

![image](https://user-images.githubusercontent.com/16143771/177665594-28149fdf-7db8-4690-a948-7f0f168ed018.png)